.travis.yml

```diff
@@ -47,3 +47,14 @@ branches:
     - /^v\d/
 notifications:
   email: false
+deploy:
+  provider: puppetforge
+  username: puppet
+  password:
+    secure: "whG1hlsH4KREOkHopzdPRc965y005VzSwTpSzFw9ho30QSvvwv/jAJ7izbkrYwaTHGlOC5S1Wk3k0UsuVp0NE69zErnOFplqHo0BLI40uuhTCQQ4Qm6i/+44AtmjtFGO//cWs4fr6yyVC9kyIvsP+3rx/qt6GNxoltbNkvb7Hco="
```

Comment:
I mistakenly closed the review, but yeah I don't think we want to tell anybody who pulls down this module what your deploy key is

Reply:
eh? This is a travis secure value that's tied to this repo.

Reply:
oh.. that's somewhat confusing to me. So is the secure value for travis or puppet forge? Also, can you confirm that there is no requirement around making sure this value is secure, stored in git, etc?

Reply:
It is a pgp encrypted secret of my forge passphrase.  Only travis has the private key and its tied to this repo. See https://docs.travis-ci.com/user/encryption-keys/

Reply:
oh perfect! Was just making sure nothing got mixed up there.